kyco/core/events.py

```diff
@@ -12,37 +15,39 @@ class KycoEvent(object):
     Args:
         content (dict): Dictionary with all event informations
     """
-    def __init__(self, content):
+    def __init__(self, content, connection=None, timestamp=datetime.now()):
```

Comment:
This should be with timezone support. For instance:
`datetime.datetime.now(timezone.utc)`